VisualPinball.Unity/VisualPinball.Unity/VPT/Kicker/KickerCollider.cs

```diff
@@ -100,9 +100,11 @@ internal static class KickerCollider
 							// Only mess with variables if ball was not kicked during event
 							ball.Velocity = float3.zero;
 							ball.AngularMomentum = float3.zero;
-							var posZ = staticData.FallThrough
-								? staticData.ZLow - ball.Radius - 5.0f
-								: staticData.ZLow + ball.Radius;
+							var posZ = !staticData.FallIn
+								? staticData.ZLow + ball.Radius * 2
+								: staticData.FallThrough
+									? staticData.ZLow - ball.Radius - 5.0f

```

Comment:
Is it worth making 5.0f a constant?

Reply:
It's like that in the VPX code, so I'd prefer to stays as close as possible.